src/twisted/python/test/test_setup.py

```diff
@@ -326,26 +329,20 @@ def test_find_package_modules(self):
                 "twisted.test.mock_win32process",
                 ],
             )
-        # We are in _trial_temp which can be at PWD/_trial_temp when executed
-        # outside of tox or some tox temporary folder.
-        # When running under tox we use TOX_INI_DIR to find where the source
-        # code is located.
-        basePath = os.environ.get('PWD', '../')
-        # Overwrite the path if we are running with tox.
-        basePath = os.environ.get('TOX_INI_DIR', basePath)
-        packageDir = os.path.join(basePath, 'src', 'twisted', 'test')
+        twistedPackageDir = filepath.FilePath(twisted.__file__).parent()
```

Comment:
We do, because people kept screwing this up, and we never got to the point where it all worked :).